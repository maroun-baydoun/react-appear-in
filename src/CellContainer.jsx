import React from "react";
import Cell from "./Cell";

const CellContainer = () => (
  <div className="cell-container">
    {[...Array(10)].map((_, i) => (
      <Cell key={i} />
    ))}
  </div>
);

export default CellContainer;
