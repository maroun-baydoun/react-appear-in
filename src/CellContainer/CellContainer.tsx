import React from 'react';
import styled from 'styled-components';
import Cell from './Cell';

const CellContainer = () => (
    <StyledContainer>
      {[...Array(100)].map((_, i) => <Cell key={i} />)}
    </StyledContainer>
  );

const StyledContainer = styled('div')`
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  background-color: grey;
`;

export default CellContainer;
