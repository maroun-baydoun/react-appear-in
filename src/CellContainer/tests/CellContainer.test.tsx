import React from 'react';
import ReactTestRenderer from 'react-test-renderer';

import CellContainer from '../CellContainer';

describe('<CellContainer/>', () => {
  it('renders component', () => {
    const renderedComponent = ReactTestRenderer.create(<CellContainer/>);
    expect(renderedComponent.toJSON()).toMatchSnapshot();
  });
});
