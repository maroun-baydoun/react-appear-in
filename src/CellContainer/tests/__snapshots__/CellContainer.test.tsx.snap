// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CellContainer/> renders component 1`] = `
<div
  className="sc-ifAKCX knZWyk"
>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eMgRRL"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cRtsZj"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa gFYrlx"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa ktYNSv"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa ctnNjG"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jXedtY"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa gCfQtx"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa iOLaPh"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jexCjL"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa kuyxKp"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hNRpyu"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa kjmpkV"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cfPPSS"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bYRKoo"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa dIaLaI"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa blXtJB"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa Asyqs"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa iuYcFo"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hBaqBH"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa dHOVNT"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eBeCXd"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cwwOkw"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa gGJyDg"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jtDlfI"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eZEtMP"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jCSaZH"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa elwizh"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jxRifh"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa buzjZe"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jgkfRW"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fEeyrX"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bMfvdk"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jhQAuT"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa imjLgG"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hOTnYx"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa koCYMt"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eaJABA"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hpznQz"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa GjtSt"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa PhQRk"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bbomJB"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa dCyJRN"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa exgQKr"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jMXjXK"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cwIBBN"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bUeidX"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa ejaZNY"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jBlfFo"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bEmYhw"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cfEeWa"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bZbShi"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa DSTOB"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa KwWgF"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa gahEku"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jxpSix"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eioMLs"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hefVSA"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cdYJpq"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa feCdWr"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hhSful"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cnfGKi"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fDGTgz"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fcappu"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hZimwN"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hOJxza"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa kQzXtp"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jTBbQx"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa kjrTaX"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa iyoAcF"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa kvJKqm"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eOqcSc"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa zVMOr"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bmhbiq"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fdFJuQ"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa lbeHHE"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jCGTqR"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hXywRq"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fHIXVa"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eYiaOw"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa jkNpNb"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fBzGpM"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa igaoSR"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa irYTBL"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hNTViP"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa gDyJRz"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa iCMTLA"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa bbTNnG"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa iVVQFh"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fBcJGP"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa dMILXT"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa gynPWA"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa dbGotk"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa cZxCAH"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa fhsWaT"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa kwxndY"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa eBiph"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa gACRrf"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa grpMoi"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa hbJVnB"
    />
  </div>
  <div
    className="sc-htpNat cdFCkK"
  >
    <div
      className="sc-bdVaJa dJbQZZ"
    />
  </div>
</div>
`;
